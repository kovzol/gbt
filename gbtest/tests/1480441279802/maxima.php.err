0.16user 0.01system 0:00.19elapsed 93%CPU (0avgtext+0avgdata 70976maxresident)k
0inputs+480outputs (0major+4333minor)pagefaults 0swaps
