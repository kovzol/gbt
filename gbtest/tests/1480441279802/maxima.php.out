
Maxima 5.37.2 http://maxima.sourceforge.net
using Lisp GNU Common Lisp (GCL) GCL 2.6.12
Distributed under the GNU Public License. See the file COPYING.
Dedicated to the memory of William Schelter.
The function bug_report() provides bug reporting information.
(%i1)                                                  2
(%o3) [x1, x2, x6, x3, (- x4 y) + y - x x5 + 2 x4  - 3 x4 + 1, 
      2                              2
2 x5 y  - 4 x5 y + 3 x x4 y - x y + x  x5 + 2 x5 + x x4 - 3 x, 
       2     2                                            2
(- x4 y ) + y  - x x5 y + 2 x4 y - 2 y - 3 x x5 - x4 + 2 x  + 1, 
       3        2         2                                              3
(- x5 y ) - x5 y  - x x4 y  + 5 x5 y - 6 x x4 y + 4 x y - 3 x5 - x x4 - x
              3      3          2    2  2       2                 2
 + 4 x, 4 x4 y  - 4 y  - 20 x4 y  - x  y  + 20 y  + 28 x4 y - 18 x  y - 28 y
                      4       2
 - 32 x x5 - 12 x4 + x  + 19 x  + 12, (- x5 y) + x4 x5 - x x4, 
               2                         2     2
x4 y - y - 2 x5  + x x5 - x4 + 1, (- x4 y ) + y  + 2 x4 y - 2 y - 4 x x5
    2              2
 + x  x4 - x4 + 3 x  + 1]
(%i4) 